src/lib/plugin_apis/lvm.api

```diff
@@ -5,13 +5,13 @@
 #ifndef BD_LVM_API
 #define BD_LVM_API
 
-#ifdef __LP64__
-// 64bit system
-#define BD_LVM_MAX_LV_SIZE (8 EiB)
-#else
-// 32bit system
-#define BD_LVM_MAX_LV_SIZE (16 TiB)
-#endif
+/**
+ * BD_LVM_MAX_LV_SIZE:
+ *
+ * Deprecated: 2.23: Use bd_lvm_get_max_lv_size function instead.
```

Comment:
I think that if you put `()` after the name of the function, it will work as a link in the documentation.